cincinnati/src/plugins/internal/graph_builder/openshift_secondary_metadata_parser/plugin.rs

```diff
@@ -7,6 +7,9 @@ use self::cincinnati::plugins::prelude_plugin_impl::*;
 use std::collections::HashSet;
 
 pub static DEFAULT_KEY_FILTER: &str = "io.openshift.upgrades.graph";
+static SUPPORTED_VERSIONS: &[&str] = &[
+    "1.0.0",
+];
```

Comment:
```suggestion
static SUPPORTED_VERSIONS: &[&str] = &["1.0.0"];
```
rustfmt insists on a single line

Reply:
grr, rustfmt, making it slightly more tedious for folks to expand this list later :p  Fixed with 3b3e674c -> 89c15ecc.